src/openzaak/components/documenten/tests/test_reserved_document.py

```diff
@@ -0,0 +1,131 @@
+# SPDX-License-Identifier: EUPL-1.2
+# Copyright (C) 2019 - 2020 Dimpact
+from datetime import date
+
+from django.test import TestCase
+
+from rest_framework.exceptions import ValidationError
+
+from openzaak.components.documenten.api.serializers import ReservedDocumentSerializer
+from openzaak.components.documenten.api.utils import generate_document_identificatie
+from openzaak.components.documenten.models import (
+    EnkelvoudigInformatieObject,
+    ReservedDocument,
+)
+from openzaak.components.documenten.tests.factories import (
+    EnkelvoudigInformatieObjectFactory,
+)
+
+
+class ReservedDocumentTests(TestCase):
```

Comment:
When the viewset and urlpattern for this endpoint have been added, these tests can be changed to make actual API calls to the endpoint, instead of calling the serializer methods directly. You can see an example of these kind of tests at `src/openzaak/components/zaken/tests/test_zaken.py` for example.

```suggestion
class ReservedDocumentTests(APITestCase):
```